docs/DeveloperGuide.adoc

```diff
@@ -767,6 +807,94 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+[discrete]
+=== Use case: Add person
+
+*MSS*
+
+1.  User requests to add a person
+2.  AddressBook shows an example of add person message
+3.  User follows the format and provide all the required fields
+4.  AddressBook adds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. User uses the wrong format.
++
+[none]
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+* 3b. User didn't fill in all the required fields
++
+[none]
+** 3b1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+
+[discrete]
+=== Use case: Find person
+
+*MSS*
+
+1.  User requests to find a person
+2.  AddressBook shows an example of find person message
+3.  User follows the format and provide the name of the person
+4.  AddressBook finds the person
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. User uses the wrong format.
++
+[none]
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+* 3b. Name given by the user is not in the AddressBook
++
+[none]
+** 3b1. AddressBook shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Change language
+
```

Comment:
I dont see any user stories corresponding to this. Is this one of your core feature? If so, you should have a user story for it.